recipes/ade/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class AdeConan(ConanFile):
+    name = "ade"
+    license = "Apache-2.0"
+    homepage = "https://github.com/opencv/ade"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Graph construction, manipulation, and processing framework"
+    topics = ("graphs", "opencv")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+    }
+
+    _cmake = None
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 11)
+        if (self.settings.compiler == "clang"
+            or self.settings.compiler == "apple-clang"
+            or self.settings.compiler == "gcc")  \
+                and self.settings.compiler.libcxx != "libstdc++11":
+            raise ConanInvalidConfiguration("libstdc++11 required")
+
+    def _configure_cmake(self):
+        if self._cmake is None:
+            self._cmake = CMake(self)
+        self._cmake.configure(source_folder=self._source_subfolder,
```

Comment:
Add a cmake wrapper, and configure that instead.

Reply:
I am not sure i understand, what would be the scope of the cmake wrapper?

Reply:
Include `conanbuildinfo.cmake`, call `conan_basic_setup` and add `source_subfolder` as a subproject.

Reply:
Sure i can do that. I had skipped that step since this project have dependencies as such.

Reply:
Actually, could you explain why this is required since this project has no dependencies.

Reply:
For MSVC, it will set the correct runtime (MT/MD/MTd/MDd).
For the libstdc++(11) c++ runtime, it will set the correct flag for the [Dual ABI](https://gcc.gnu.org/onlinedocs/libstdc++/manual/using_dual_abi.html) (` _GLIBCXX_USE_CXX11_ABI`).

Reply:
@madebr have updated the recipe as you suggested.